src/screens/QuestionScreen/QuestionScreen.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity, SafeAreaView, TextInput, Alert, Linking } from 'react-native';
+import { firebase } from '../../config/config';
+import { LOGO } from '../../config/styles.js';
+import styles from './styles';
+import moment from 'moment';
+import Icon from 'react-native-vector-icons/Ionicons';
+import ImagePicker from 'react-native-image-crop-picker';
+import * as Progress from 'react-native-progress';
+
+class QuestionScreen extends  React.Component {
+    state = {QueryInput : '', setimage : null, setDisplayImage: null, transferred: 0};
+
+    takephotofrommlib = () => {
```

Comment:
See the method names here, three methods are having three different naming convention methods. I did not go this deeper in other PR review, thought you might have familiar about these.

Reply:
I'll keep that in mind! Changed variable and method naming convention to camelCase type.